lib/publiccloud/utils.pm

```diff
@@ -193,9 +193,9 @@ sub is_hardened() {
 sub is_embargo_update {
     my ($incident, $type) = @_;
     return 0 if ($type =~ /PTF/);
-    script_retry("curl -sSf https://build.suse.de/attribs/SUSE:Maintenance:$incident -o /tmp/$incident.txt");
-    return 1 if (script_run("grep 'OBS:EmbargoDate' /tmp/$incident.txt") == 0);
-    return 0;
+    my $out script_output("curl -sSf https://smelt.suse.de/api/v1/basic/incidents/$incident/ | jq .embargo");
```

Comment:
```suggestion
    my $out = script_output("curl -sSf https://smelt.suse.de/api/v1/basic/incidents/$incident/ | jq .embargo");
```

Reply:
I switched to the draft , this change was added to debug why initial version of the code is not working , please ignore any further changes until I will not revert it back from Draft